query_modules/schema.cpp

```diff
@@ -152,20 +144,23 @@ void Schema::NodeTypeProperties(mgp_list * /*args*/, mgp_graph *memgraph_graph,
       }
 
       if (node_types_properties.find(labels_set) == node_types_properties.end()) {
-        node_types_properties[labels_set] = PropertyInfo{std::set<Property, PropertyComparator>(), true};
+        node_types_properties[labels_set] = LabelsInfo();
```

Comment:
I think you don't need this line, as node_types_properties[labels_set] will construct if the entry doesn't exist: https://godbolt.org/z/9TzdEj1xK

Reply:
And link to cpp ref: https://en.cppreference.com/w/cpp/container/unordered_map/operator_at